pkg/minikube/cruntime/cruntime.go

```diff
@@ -47,6 +48,11 @@ func (cs ContainerState) String() string {
 	return [...]string{"all", "running", "paused"}[cs]
 }
 
+// supportedContainerRuntimes lists the supported container runtimes
+func supportedContainerRuntimes() []string {
```

Comment:
Make this public and in Cobra's flag for --contaired runtime reference it.
Change the help text for --contaired-runtime to reference this. For example: 
Valid options are cruntime.ValidOptions()

Reply:
here is an example of cobra help
https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/start_flags.go#L130

Reply:
Implemented this in the latest commit.